src/queries/full-text-queries/query-string-query.js

```diff
@@ -303,6 +309,42 @@ class QueryStringQuery extends QueryStringQueryBase {
         this._queryOpts.escape = enable;
         return this;
     }
+
+    /**
+     * Sets the type of query string query. Valid values are:
+     * - `best_fields` - (default) Finds documents which match any field,
+     * but uses the `_score` from the best field.
+     *
+     * - `most_fields` - Finds documents which match any field and combines
+     * the `_score` from each field.
+     *
+     * - `cross_fields` - Treats fields with the same `analyzer` as though
+     * they were one big field. Looks for each word in *any* field
+     *
+     * - `phrase` - Runs a `match_phrase` query on each field and combines
+     * the `_score` from each field.
+     *
+     * - `phrase_prefix` - Runs a `match_phrase_prefix` query on each field
+     * and combines the `_score` from each field.
+     *
+     * - `bool_prefix` - Creates a match_bool_prefix query on each field and
```

Comment:
I think this type was not available in Elasticsearch v6 - https://www.elastic.co/guide/en/elasticsearch/reference/6.8/query-dsl-multi-match-query.html#multi-match-types

Could we add a note regarding the same? If you can identify the exact version that it was added in, that would be great.